apps/cmp.c

```diff
@@ -459,6 +472,8 @@ const OPTIONS cmp_options[] = {
     OPT_SECTION("Credentials format"),
     {"certform", OPT_CERTFORM, 's',
      "Format (PEM or DER) to use when saving a certificate to a file. Default PEM"},
+    {"crlform", OPT_CRLFORM, 's',
+     "Format (PEM or DER) to use when saving a CRL to a file. Default DER"},
```

Comment:
As above in regards to the default.

Reply:
Above, I have documented the reason for choosing DER as default and also added it in `openssl-cmp.pod.in`.